pep-9999.rst

```diff
@@ -0,0 +1,1037 @@
+
+If the file does not parse as a core metadata file, then the
+installer should ignore the parse failure and instead just
+use a pre-defined error message of its own, which should
+suggest that the user create a virtual environment to
+install packages.
+
+Software distributors who have a non-Python-specific package
+manager that manages libraries in the ``sys.path`` of their
+Python package should, in general, ship a
+``EXTERNALLY-MANAGED`` file in their standard library
+directory. For instance, Debian may ship a file in
+``/usr/lib/python3.9/EXTERNALLY-MANAGED`` consisting of
+something like
+
+::
+
+    To install Python packages system-wide, try apt install
+    python3-xyz, where xyz is the package you are trying to
+    install.
+
+    If you wish to install a non-Debian-packaged Python
+    package, create a virtual environment using python3 -m
+    venv path/to/venv. Then use path/to/venv/bin/python and
+    path/to/venv/bin/pip. Make sure you have python3-full
+    installed.
+
+    If you wish to install a non-Debian packaged Python
+    application, it may be easiest to use pipx install xyz,
+    which will manage a virtual environment for you. Make
+    sure you have pipx installed.
+
+    See /usr/share/doc/python3.9/README.venv for more
+    information.
+
+which provides useful and distribution-relevant information
+to a user trying to install a package.
+
+In certain contexts, such as single-application container
+images that aren't updated after creation, a distributor may
+choose not to ship an ``EXTERNALLY-MANAGED`` file, so that
+users can install whatever they like (as they can today)
+without having to manually override this rule.
+
+Writing to only the target ``sysconfig`` scheme
+-----------------------------------------------
+
+Usually, a Python package installer installs to directories
+in a scheme returned by the ``sysconfig`` standard library
+package.  Ordinarily, this is the scheme returned by
+``sysconfig.get_default_scheme()``, but based on
+configuration (e.g. ``pip install --user``), it may use a
+different scheme.
+
+Whenever the installer is installing to a ``sysconfig``
+scheme, this PEP specifies that the installer should never
+modify or delete files outside of that scheme. For instance,
+if it's upgrading a package, and the package is already
+installed in a directory outside that scheme (perhaps in a
+directory from another scheme), it should leave the existing
+files alone.
+
+If the installer does end up shadowing an existing
+installation during an upgrade, we recommend that it
+produces a warning at the end of its run.
+
+If the installer is installing to a location outside of a
+``sysconfig`` scheme (e.g., ``pip install --target``), then
+this subsection does not apply.
+
+Recommendations for distributions
+=================================
+
+This section is non-normative. It provides advice we think
+is a good idea for distributions to follow unless they have
+a specific reason otherwise.
```

Comment:
```suggestion
This section is non-normative. It provides best practices
we believe distributions should follow unless they have
a specific reason otherwise.
```